mopidy_spotify/search.py

```diff
@@ -55,11 +55,18 @@ def search(config, session, requests_session,
             'to at most 50.')
         search_count = 50
 
+    params = {
+        'q': sp_query,
+        'limit': search_count,
+        'type': ','.join(types)}
+
+    try:
+        params['market'] = session.user_country
+    except ValueError as exc:
```

Comment:
I guess if we were really worried about libspotify returning non utf-8 bytes back to us we should handle it in pyspotify. I can remove this (and it's test).